test/jdk/javax/swing/JPopupMenu/4966112/bug4966112.java

```diff
@@ -109,6 +131,7 @@ public static void main(String[] args) throws Exception {
         robot.waitForIdle();
         closeFrame();
 
+        isAquaFileChooser = false;
```

Comment:
Why this line at the end of the test is needed?

Reply:
It is needed if someone adds some other test cases to the end of test.

Reply:
I doubt that if some test will be added later the filechooser under the test became non-aqua.

Reply:
It is the Aqua from the beginning till the end or some other L&F

Reply:
Ok, in that case we can remove this line